lib/caesar.rb

```diff
@@ -1,6 +1,33 @@
 class Caesar
-  attr_reader :key
+
+  attr_reader :message,
+              :rotation
 
   def initialize
   end
-end
\ No newline at end of file
+
+  def eng_to_cipher(message, rotation)
+    @message = message
```

Comment:
I wouldnt use ivars here, you should just use local variables to do the work here.